### PR TITLE
[14.0][MIG] report_qweb_encrypt

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready fo review.

### CODE REVIEW

File: report_qweb_encrypt/static/src/js/report/action_manager_report.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+odoo.define("report_qweb_encrypt.Dialog", function (require) {
+    "use strict";
+
+    var ActionManager = require("web.ActionManager");
+    var Dialog = require("web.Dialog");
+    var core = require("web.core");
+
+    var _t = core._t;
+
+    var EncryptDialog = Dialog.extend({
```
Comment (kittiu) [inline report_qweb_encrypt/static/src/js/report/action_manager_report.js:12]: I still can't get the password dialog to popup. I have zero knowledge about JS. Can someone take a look?

Comment (kittiu) [reply report_qweb_encrypt/static/src/js/report/action_manager_report.js:12]: Fixed by this, something changed from v12.

https://github.com/OCA/reporting-engine/compare/104472445acef310d720c25acd1ff934061b40dd..6e79bfd5cf80bd41a617931fb1f96651789ba1f9

**Review by marcelsavegnago — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Can you rebase like you rebased the 12.0 version?

**Comment by kittiu** [comment]
> Can you rebase like you rebased the 12.0 version?

Thank you @StefanRijnhart should be ready now.

**Comment by StefanRijnhart** [comment]
Thank you. Travis fails on missing wkhtmltopdf. It's installation is likely triggered by setting the version as a variable as is done here for 13.0: https://github.com/OCA/reporting-engine/blob/13.0/.travis.yml#L44

File: report_qweb_encrypt/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Authors
+~~~~~~~
+
+* Creu Blanca
+
+Contributors
+~~~~~~~~~~~~
+
+* Enric Tobella <etobella@creublanca.es>
+* Jaime Arroyo <jaime.arroyo@creublanca.es>
```
Comment (StefanRijnhart) [inline report_qweb_encrypt/README.rst:63]: ~~You accidentally removed yourself as a contributor!~~ Oh it's just in the compiled README.

**Comment by kittiu** [comment]
> Thank you. Travis fails on missing wkhtmltopdf. It's installation is likely triggered by setting the version as a variable as is done here for 13.0: https://github.com/OCA/reporting-engine/blob/13.0/.travis.yml#L44

Thanks!

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**